### PR TITLE
add page 404.html

### PR DESCRIPTION
**Pull request by IvanTymoshchuk**

### CODE REVIEW

File: frontend/public/404.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Marco Ramirez</title>
```
Comment (annaBalabak) [inline frontend/public/404.html:5]: correct the title

File: frontend/public/404.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    <script type="text/javascript">
+      // MIT License
```
Comment (annaBalabak) [inline frontend/public/404.html:7]: delete comments

File: frontend/public/index.html [hunk 3]
```diff
@@ -30,6 +30,22 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+     <script type="text/javascript">
+      // MIT License
```
Comment (annaBalabak) [inline frontend/public/index.html:34]: Do we need these comments?

**Comment by Svitlana-G** [comment]
How does a page 404 work?
 I added to body  in 404.html:   "Page not found" (in h1) and went to a non-existent page http://localhost:3000/something.html
 As a result I received 
 
image

And I can't see "Page not found" on this page.

**Pull request closed without merging**